src/explore-education-statistics-admin/src/config/ckEditorConfig.ts

```diff
@@ -104,3 +105,5 @@ export const tableContentToolbar: string[] = [
   'tableRow',
   'mergeTableCells',
 ];
+
+export const alignmentOptions: string[] = ['left', 'right', 'center'];
```

Comment:
Just wondering if you've thought about potentially narrowing this type? I believe the following is the correct interface: 
```
AlignmentConfig
```

Reply:
Good idea done now.